src/Library/demos/Power Profile Monitor/main.py

```diff
@@ -0,0 +1,27 @@
+import gi
+
+gi.require_version("Adw", "1")
+from gi.repository import Adw, Gio
+import workbench
+
+power_profile_monitor = Gio.PowerProfileMonitor.dup_default()
+overlay = workbench.builder.get_object("overlay")
+
+
+def on_power_saver_enabled(monitor):
+    if power_profile_monitor.get_power_saver_enabled():
```

Comment:
Sorry, I'll fix that real fast.

Reply:
Well, the way the JS demo does it is practically doing `lambda *_:` (but in JS) and then the function, using `power_profile_monitor`. Python lambda can't handle functions longer than one line, so I made a separate function. And since I was writing a function specifically to be used as a callback, I though I might as well accept the arguments are passed to the callback.

What do you think, should I follow the JS demo more closely and ignore the arguments, or should I handle the passed arguments?

Reply:
Well, in https://github.com/sonnyp/Workbench/pull/724/files it's sort of a mess of both :grimacing:

Reply:
I would suggest this:
- If it's a lambda, it's OK to use the global if this is also what the JS code does
- If it's a function and it gets the widget itself as an argument, it should use the argument.

But in any case we must be consistent in one demo and if you do name the argument (instead of using *args) please also use it instead of the global.

Reply:
Also we should not shadow global variables. I'll write these things down for a future linter...